manifests/20-performance-profile.crd.yaml

```diff
@@ -382,6 +382,15 @@ spec:
                     reserved:
                       description: Reserved defines a set of CPUs that will not be used for any container workloads initiated by kubelet.
                       type: string
+                    workloads:
```

Comment:
Why is this in the cpu section? All the other fields here accept cpu ids, not annotation names. I think this will confuse users.

Reply:
Resolving: Changed approach no longer includes PA API Change